### DISCUSSION TITLE
Can we have an introduction to "themes"?

### DISCUSSION THREAD

**Discussion by dmalawey** [upscayl/upscayl#534] [Q&A] [answered]
Hello, I'm trying Upscayl for the first time (version 2.8.6, desktop, PC)

### Can we add an explanation for the Themes found in settings?
* I searched [the Wiki](https://github.com/upscayl/upscayl/wiki) for explanation of Themes but couldn't find one.

### Examples:
* Perhaps a photo with thumbnails of processing with multiple themes - this may be the fastest way to inform noobs.

### My use case:
* Today I'm looking to improve an [old business card ](https://i.imgur.com/h5BdmT8.jpg)(from my grandfather's garage) found here, so I can archive a good quality copy of it.
* From the start, I don't know if Upscayl is appropriate for this kind of document-restoration, but I really love the open source and high quality app, so I want to support it and keep the application even if it's not the right tool for this job.

**Comment by aaronliu0130** [comment, marked as answer]
Themes just change your UI. They have no effect on the image processing. 

In the future, when in doubt about an option, just change it a bit and see what happens.
